src/darker/formatters/__init__.py

```diff
@@ -0,0 +1,38 @@
+"""Built-in code re-formatter plugins."""
+
+from __future__ import annotations
+
+import sys
+from importlib.metadata import EntryPoint, entry_points
+from typing import cast
+
+from darker.formatters.base_formatter import BaseFormatter
+
+ENTRY_POINT_GROUP = "darker.formatter"
+
+
+def get_formatter_entry_points(name: str | None = None) -> tuple[EntryPoint, ...]:
+    """Get the entry points of all built-in code re-formatter plugins."""
```

Comment:
Sorry probably missing something, but are we registereing all entry points for all currently installed formatters?
Would this register black and ruff?
Are we not supporting that below 3.10?